packages/block-library/src/list/ordered-list-settings.js

```diff
@@ -24,6 +29,21 @@ const OrderedListSettings = ( { setAttributes, reversed, start } ) => (
 				value={ Number.isInteger( start ) ? start.toString( 10 ) : '' }
 				step="1"
 			/>
+			<SelectControl
+				__nextHasNoMarginBottom
+				label={ __( 'Numbering style' ) }
+				options={ [
+					{ value: '1', label: __( 'Numbers' ) },
+					{ value: 'A', label: __( 'Uppercase letters' ) },
+					{ value: 'a', label: __( 'Lowercase letters' ) },
+					{ value: 'I', label: __( 'Uppercase roman numerals' ) },
+					{ value: 'i', label: __( 'Lowercase roman numerals' ) },
```

Comment:
"Roman" is a proper noun

```suggestion
					{ value: 'i', label: __( 'Lowercase Roman numerals' ) },
```